Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,96 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render an `<picture>` element for responsive image support.
```

Comment:
Minor typo here: Should be "render a `<picture>`"

Reply:
Yep. I copied code from somewhere, and then modified it. Whatever it replaced was something than needed an 'an' and I did not notice it. Ooops.